### ISSUE TITLE
"book imported" screen reader announcement missing (tested in JAWS)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#762]
@mutterback says:
https://github.com/readium/readium-desktop/pull/758#issuecomment-538382881

Announcements that a new book is imported by eKitabu are not announced in JAWS (only tested JAWS)

**Comment by danielweck** [comment]
This works fine in Thorium 2.1.0 (toast ARIA live assertive notification)
